src/core/frame.h

```diff
@@ -164,6 +164,10 @@ typedef enum QUIC_FRAME_TYPE {
 
 } QUIC_FRAME_TYPE;
 
+CXPLAT_STATIC_ASSERT(
+    QUIC_FRAME_MAX_SUPPORTED <= (uint64_t)UINT16_MAX,
+    "Sent packet metadata 'Type' field above assumes frames types fit in 16-bits");
```

Comment:
replace "Sent packet metadata" with `QUIC_SENT_FRAME_METADATA`
remove "above", since the referred-to field is not defined above in this file.

Reply:
I want to keep both, because they are pointing to two separate requirements that needs to be met.